docs/plotting.rst

```diff
@@ -41,9 +41,9 @@ We train on a 150 random points in :math:`[0, 1]`.
 
 >>> import numpy
 >>> from fuel.streams import DataStream
->>> from fuel.datasets import ContainerDataset
+>>> from fuel.datasets import IterableDataset
 >>> sample = theano.tensor.scalar('data')
->>> data_stream = DataStream(ContainerDataset(
+>>> data_stream = DataStream(IterableDataset(
```

Comment:
Perhaps I a bit late with this, but I can't help reading `IterableDataset` as "a dataset that supports iteration" and not "dataset made from iterables"